dash.el

```diff
@@ -1048,6 +1048,21 @@ This function can be thought of as a generalization of
   "Return a list of ((-filter PRED LIST) (-remove PRED LIST)), in one pass through the list."
   (--separate (funcall pred it) list))
 
+(defun -separate-multi (preds list)
+  "Return a list of ((-filter PRED1 LIST) (-filter PRED2 LIST) ... REST), in one
+  pass through the list. PREDS is a list of functions and REST are elements of
+  LIST that are unmatched by any PREDS."
```

Comment:
Thanks for this, I just added more commits to fix this. Do you think I should take out the `((-filter PRED1 LIST)...)` part? Is it confusing because `REST` is not a parameter? If I leave that in I can't fit in `one pass through the list` on the first line. As it is I separated it into it's own line.